node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
@rwaldron I suggested @HipsterBrown make a function b/c you can only set the state with a function and the only way to get the state is via a property (`led.value`). Seems inconsistent that there would be no function for getting the state.